nodes/liveplot

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
+"""
+plots ros superpackets in 3D
+"""
+import matplotlib.pyplot as plt
+import mpl_toolkits.mplot3d.axes3d as p3
+import matplotlib.animation as animation
+import roslib
+roslib.load_manifest('strawlab_freeflight_experiments')
+import rospy
+import ros_flydra.msg
+import argparse
+import collections
+
+rospy.init_node("followpath_monitor_trace")
+
+
+class rosHandler():
+    def __init__(self):
+        self._super_packet = None
+        rospy.Subscriber("/flydra_mainbrain/super_packets",
+                         ros_flydra.msg.flydra_mainbrain_super_packet,
+                         self._on_target_move)
+    def _on_target_move(self,msg):
+        self._super_packet = msg.packets[0]
+    def get_super_packet(self):
+        return self._super_packet
+
+class DataHandler():
+    '''
+    we store xzy for every objid with this class to a circular buffer; new objIds are added, old 
+    ones that are not updated for a periode of time are deleted 
+    '''
+    def __init__(self, buffer_length=1000, dead_after_secs=30):
+        self.obj = {} #memory
+        self.buffer_length = buffer_length
+        self.dead_after_secs = dead_after_secs
+        self.call_ros = rosHandler()
```

Comment:
there isn't really a need to have a separate ros object (as this class also has explicit knowledge of ros things - like the message structure. Just put the subscriber here